GVFS/GVFS.Common/Maintenance/LooseObjectsStep.cs

```diff
@@ -12,7 +12,8 @@ namespace GVFS.Common.Maintenance
     public class LooseObjectsStep : GitMaintenanceStep
     {
         public const string LooseObjectsLastRunFileName = "loose-objects.time";
-        private readonly bool forceRun;
+        public int LooseObjectsPutIntoPackFile = 0;

```

Comment:
`{ get; private set; }`. Initializes to zero anyway, but want to avoid public setters.